src/js/actions/LibraryActions.js

```diff
@@ -71,9 +71,14 @@ export default {
             properties: ['openDirectory', 'multiSelections']
         }, (folders) => {
             if(folders !== undefined) {
-                store.dispatch({
-                    type : AppConstants.APP_LIBRARY_ADD_FOLDERS,
-                    folders
+                const realpathAsync = Promise.promisify(require('fs').realpath);
```

Comment:
Can you assign this at the top level scope?
Also, you don't need to require fs, you can just use imported variable. (It is ok to use it after it is imported)